server/routerlicious/packages/services-core/src/tenant.ts

```diff
@@ -86,6 +86,11 @@ export interface ITenantManager {
      */
     getTenant(tenantId: string, documentId: string): Promise<ITenant>;
 
+    /**
+     * Retrieves GitManager instance for the given tenant
+     */
+    getTenantGitManager(tenantId: string, documentId: string): Promise<GitManager>;
```

Comment:
Since this is an interface it would be better to IGitManager instead of GitManager